website/docs/release_notes.md

```diff
@@ -2,9 +2,95 @@
 title: Release notes
 sidebar_position: 99
 ---
+import ReactPlayer from 'react-player'
+
 
 # Release notes
 
+## [v0.1.19](https://github.com/VirtusLab/scala-cli/releases/tag/v0.1.19)
+
+## What's Changed
+
+## Fix `workspace/reload` Intellij IDEA
+
+Pick up dependencies from `using` directives after re-importing a BSP project.
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/18601388/207319736-534f2d8a-862d-4c0a-8c8a-e52d95ac03e6.mov' />
+
+Fixed by [@Gedochao](https://github.com/Gedochao) in [1681](https://github.com/VirtusLab/scala-cli/pull/1681).
+
+##  `shebang` headers in Markdown
+
+The `shebang` headers in `scala` code blocks inside a markdown input are always ignored.
```

Comment:
NIT I'm not sure if this is worth highlighting... IMO might as well put it in the `Fixes` section (or just leave it as is, I dunno).